salt/mirror/minima_aws_testsuite.yaml

```
@@ -0,0 +1,318 @@
+storage:
```

What is the need to have this file different from the default one?

---

This file exists, for now, so we don't have on AWS the following repositories:
- All that is "Maintenance:" 
- SLES15SP2 repositories 

As we talked, for now we don't want to have those packages on AWS MIrror. In future if is ok to have, we can just remove this file.

---

How about using some Jinja templating, or just leaving them there (they should just fail and not really hurt)?

---

The goal was to not have these repositories on AWS when we sync from a local mirror to aws. So this file should be used to the local mirror.
This is expect to be something temporary until it's get clarified if we can have those repositories on cloud.
Do you think we should have the effort to use jinja Template for it or just leave it there?

---

I would even let them there, internal repos will simply silently fail to sync and we do not get the extra overhead of having templating. Only if this becomes a problem I'd try to address it.

---

agree. I will remove the duplicated file.